cli.js

```diff
@@ -8,6 +8,7 @@ var fs = require('fs');
 var utils = require('./utils');
 
 var mbtiles = argv._;
+var accessToken = argv.MapboxAccessToken || process.env.MapboxAccessToken;
```

Comment:
In Mapbox GL native we set a MAPBOX_ACCESS_TOKEN environment variable.  Perhaps also include that as discussed in https://github.com/mapbox/mapbox-cli-py/issues/53?